### DISCUSSION TITLE
[FIXED] How I can preload a world?

### DISCUSSION THREAD

**Discussion by FlxwDNS** [Minestom/Minestom#1477] [Help] [answered]
Hello, if I join my Server first time after Start it takes long to load...
But can i preload the world? I joined when the server finished loading.

https://user-images.githubusercontent.com/57632174/197576971-f6f7c5d6-36dc-414b-a83d-f76988915d88.mp4

**Comment by emortaldev** [comment]
you can preload with `instance.loadChunk(x, z)`
also you seem to just have slow everything lol

**Reply by FlxwDNS** [reply]
But why it loades so long?

**Reply by ZakShearman** [reply]
Run a profiler to help determine this. I do experience what I would describe as long world loading times, but nothing at all like this.

**Reply by FlxwDNS** [reply]
How?

**Reply by FlxwDNS** [reply]
@emortaldev But I want to load the whole world at startup

**Reply by emortaldev** [reply]
> @emortaldev But I want to load the whole world at startup

Then preload lots of chunks

**Reply by FlxwDNS** [reply]
but how i can preload all chunks?

**Comment by FlxwDNS** [comment, marked as answer]
I have the fix! 

Put this under `minecraftServer.start("127.0.0.1", 25565);`:

```
        FakePlayer.initPlayer(UUID.randomUUID(), "ChunkLoader", fakePlayer -> {
            fakePlayer.setNoGravity(true);
            fakePlayer.setAutoViewable(false);
            fakePlayer.getOption().setInTabList(false);
        });
 ```
